app/frontend/users/UserEdit.res

```diff
@@ -285,6 +335,12 @@ let make = (
   }
 
   let (state, send) = React.useReducer(reducer, initialState)
+
+  let handleUpdateEmailFormSubmit = _ => {
+    send(SetDisableUpdateEmail(false))
+    updateEmail(send, email, state.email)
+  }
```

Comment:
Why is this function required? 
The submit button can directly call `updateEmail`
```suggestion
```